consensus/types/src/execution_witness.rs

```diff
@@ -0,0 +1,143 @@
+use crate::{test_utils::TestRandom, *};
+use derivative::Derivative;
+use serde_derive::{Deserialize, Serialize};
+use ssz_derive::{Decode, Encode};
+use test_random_derive::TestRandom;
+use tree_hash_derive::TreeHash;
+
+#[derive(
+    Default, Debug, Clone, Serialize, Deserialize, Encode, Decode, TreeHash, TestRandom, Derivative,
+)]
+#[derivative(PartialEq, Hash(bound = "T: EthSpec"))]
+#[serde(bound = "T: EthSpec")]
+#[ssz(struct_behaviour = "transparent")]
+#[serde(transparent)]
+pub struct BandersnatchGroupElement<T: EthSpec> {
```

Comment:
If you're using these transparent structs just for readability, you could also do something like:

```rust
pub type BandersnatchGroupElement<T> = FixedVector<u8, T>;
```